src/content/pages/darksouls3/aldrich-devourer-of-gods.mdoc

```diff
@@ -55,71 +59,71 @@ http:///File:Aldrich_horizontal_into_plunge.mp4
 
-- This one cycle is already tight with the stats and upgrades used in [Any%\_No_TearDrop](/darksouls3/any-no-teardrop), meaning _patch 1.08 Carthus Rouge, 43 Dexterity, +6 Sharp Sellsword Twinblades._, and will be difficult or impossible to pull off in other categories if you do not have similar damage levels.
+- This one cycle is already tight with the stats and upgrades used in [Any% Glitchless](/darksouls3/any-glitchless), meaning *patch 1.08 Carthus Rouge, 43 Dexterity, +6 Sharp Sellsword Twinblades.*, and will be difficult or impossible to pull off in other categories if you do not have similar damage levels.
   - For this opener, stand far back and roll backwards, then get 4 body/head hits in.
-  - Walk slightly back, baiting _Aldrich_ to queue up another attack, then get 1 hit in as he winds up the attack.
+  - Walk slightly back, baiting *Aldrich* to queue up another attack, then get 1 hit in as he winds up the attack.
   - Roll backwards, ideally not baiting him in a direction where he will end up in a pillar, then get 4x body hits.
   - Regenerate enough stamina for 3 hits, then finish him off before he teleports.
 
 ### **Poke (two cycle)**
 
-http:///File:Aldrich_poke_two_cycle.mp4
+{% youtube src="https://www.youtube.com/watch?v=HQD8yRqE5Fg" /%}
 
-_Note that you are able to get 1 hit in as he is winding up the poke opener, given correct stamina management and timing._
+*Note that you are able to get 1 hit in as he is winding up the poke opener, given correct stamina management and timing.*
 
 - For this opener, roll backwards and get 4 body/head hits in.
 - Stay close and regenerate stamina for 3 hits, hit him 3 more times while tanking the aoe, then get 1 more hit in as he teleports.
 - Heal and prepare for the teleport.
 
 ## **P2 attacks**
 
-_The number of hits required to finish aldrich in the second phase heavily relies on what opener attack you were given, and so these videos cannot show every instance._ *These videos are made with roughly the damage dealt in [Any%\_No_TearDrop](/darksouls3/any-no-teardrop) given the *Arrows* opener or slightly less than it, and so if you deal less damage than what is shown in that clip, you will need to get more hits in, which may mean not killing **Aldrich** in a single attack in phase 2.*
+*The number of hits required to finish aldrich in the second phase heavily relies on what opener attack you were given, and so these videos cannot show every instance.*  *These videos are made with roughly the damage dealt in* [*Any%\_No\_TearDrop*](/darksouls3/any-no-teardrop) *given the Arrows opener or slightly less than it, and so if you deal less damage than what is shown in that clip, you will need to get more hits in, which may mean not killing* ***Aldrich*** *in a single attack in phase 2.*
```

Comment:
I haven't messed around with wiki enough to know, but I noticed that some links are written as `Any%_No_TearDrop` and other as `Any%\_No\_TearDrop`. I assume they work the same but worth it to point it out in case we notice something funky in prod.